### ISSUE TITLE
release 2.5.11

### ISSUE THREAD

**Issue by slsevilla** [CCBR/RENEE#95]
# Steps
- [x] Run test data through main branch on biowulf
- [x] Run test data through main branch on FRCE
- [x] Most recent commit passes **Tests / Dryrun_Lint** github actions workflow.

# Test: Biowulf
Deployment:
```
RENEE_DIR=/data/sevillas2/RENEE
WORK_DIR=/data/sevillas2/renee_2.5.11_biowulf
if [[ -d $WORK_DIR ]]; then rm -rf $WORK_DIR; fi
$RENEE_DIR/bin/renee run \
    --input $RENEE_DIR/.tests/*.gz \
    --output $WORK_DIR \
    --genome hg38_30 \
    --mode slurm \
    --sif-cache /data/CCBR_Pipeliner/SIFS
```
Status:
- 1/16 In progress (Successfully submitted master job: 17190892)
- 1/17 completed successfully

# Test: FRCE
Deployment:
```
RENEE_DIR=/mnt/projects/CCBR-Pipelines/pipelines/RENEE/v2.5.10
WORK_DIR=/scratch/cluster_scratch/$USER/renee_2.5.11_frce
if [[ -d $WORK_DIR ]]; then rm -r $WORK_DIR; fi
module load singularity
$RENEE_DIR/bin/renee run \
    --input $RENEE_DIR/.tests/*.gz \
    --output $WORK_DIR \
    --genome hg38_30 \
    --mode slurm \
    --sif-cache /mnt/projects/CCBR-Pipelines/SIFs
```
Status:
- In progress (Successfully submitted master job:30434596)

**Comment by slsevilla** [comment]
FRCE run failed for me, with multiple rules failing due to this error: 
```
FATAL:   container creation failed: mount /lscratch->/lscratch error: while mounting /lscratch: mount source /lscratch doesn't exist
```

Unsure why the pipeline is not detecting FRCE and appears to think it's still on Biowulf. @kelly-sovacool is running a test on FRCE to determine if it's a user issue or if it's something more systemic.

**Comment by kelly-sovacool** [comment]
> FRCE run failed for me, with multiple rules failing due to this error:
> 
> ```
> FATAL:   container creation failed: mount /lscratch->/lscratch error: while mounting /lscratch: mount source /lscratch doesn't exist
> ```
> 
> Unsure why the pipeline is not detecting FRCE and appears to think it's still on Biowulf. @kelly-sovacool is running a test on FRCE to determine if it's a user issue or if it's something more systemic.

My test on FRCE worked 🤔

```sh

RENEE	Fri Jan 19 11:26:54 EST 2024
Running pipeline with the following parameters:
	b	/mnt/projects/CCBR-Pipelines,/mnt/projects/CCBR-Pipelines/Pipelines/RENEE/renee-dev-sovacool/.tests,/scratch/cluster_scratch/sovacoolkl/renee_test_v2.5.11,/scratch/cluster_scratch/sovacoolkl
	c	/scratch/cluster_scratch/sovacoolkl/renee_test_v2.5.11/.singularity
	e	slurm
	h	frce
	j	pl:renee
	o	/scratch/cluster_scratch/sovacoolkl/renee_test_v2.5.11
	t	/scratch/cluster_scratch/sovacoolkl/renee_test_v2.5.11
	w	--nowait
[Fri Jan 19 11:26:55 EST 2024] Pipeline submitted to cluster.
Master Job ID: 30547456
JobID         Partition  Timelimit    Elapsed        NodeList              Submit               Start                 End      State ExitCode
------------ ---------- ---------- ---------- --------------- ------------------- ------------------- ------------------- ---------- --------
30547456           norm 4-00:00:00   00:49:17           cn062 2024-01-19T11:26:55 2024-01-19T11:27:00 2024-01-19T12:16:17     FAILED      1:0
```

**Comment by kelly-sovacool** [comment]
Oh, it's because I set `--tmp-dir`. I'll work on #79 for this release

**Comment by slsevilla** [comment]
issue with permissions #101 fixed. FRCE run completed successfully!

**Comment by kelly-sovacool** [comment]
closed by #102
